sample/GoogleIO_17/event

```diff
@@ -0,0 +1,68 @@
+{
+    "id": 173, 
+    "name": "Google IO 2017", 
+    "latitude": 0.0, 
+    "longitude": 0.0, 
+    "location_name": "Shoreline Amphitheatre, Mountain View, CA,", 
+    "start_time": "2017-05-17T10:00:00", 
+    "end_time": "2017-05-19T17:00:00", 
+    "timezone": "UTC", 
+    "description": "<p></p>", 
+    "background_image": "/images/background.png.png", 
+    "logo": "/images/logo.png.png", 
```

Comment:
We load the logo image from API if provided in MainActivity, so that should be fixed. Check why it isn't being loaded. 

Please take this issue to Android Repo